ui/apps/platform/src/Containers/NetworkGraph/components/ClusterSelector.tsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import { Select, SelectOption } from '@patternfly/react-core';
 
-import { Cluster } from 'types/cluster.proto';
+import { ScopeObject } from 'services/RolesService';
 import useSelectToggle from 'hooks/patternfly/useSelectToggle';
 import { ClusterIcon } from '../common/NetworkGraphIcons';
 
-type ClusterSelectorProps = {
-    clusters: Cluster[];
+export type ClusterSelectorProps = {
+    clusters: ScopeObject[];
```

Comment:
Yann added the ScopeObject after the Tech Preview launch of NG 2.0, but this really highlights the ambiguous nature of that name.

I see that it's also used for Namespaces in another context.

Worth considering making type aliases for "ClusterScopesObject" and "NamespaceScopeObject", and other things, in the future.

Reply:
I agree and wasn't really thrilled with the ambiguity of this change either, but thought it was important so that we don't rely on properties of `Cluster` downstream that don't really exist.

I'd like to add the type aliases now since we are already revisiting the code, might as well make this more readable.